Dockerfile.rh.centos

```diff
@@ -75,4 +92,4 @@ USER nexus
 ENV INSTALL4J_ADD_VM_PARAMS="-Xms2703m -Xmx2703m -XX:MaxDirectMemorySize=2703m -Djava.util.prefs.userRoot=${NEXUS_DATA}/javaprefs"
 
 ENTRYPOINT ["/uid_entrypoint.sh"]
-CMD ["sh", "-c", "${SONATYPE_DIR}/start-nexus-repository-manager.sh"]
+CMD ["/opt/sonatype/nexus/bin/nexus", "run"]
```

Comment:
If you're using "nexus run" to start up the server, do we need to establish that "# Legacy start script" above?

Reply:
We found one of our internally derived images invoked the script, so its possible external folks may be doing that also.

Reply:
Ah. good catch. We wouldn't want to kill it and break other usages.  I might have been tempted to continue to CMD the script, since we know it's there. 🤷